CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+
+# v0.33.2 (2018-09-09)
+- Added `inline` and `content_unformatted` options from `js-beautify` html settings and cleared `unformatted`.  This is a breaking change, but a general improvement in the behavior to more accurately beautify html.
```

Comment:
Could you add a link to this PR and any PRs or Issues from `js-beautify`? And maybe bold `breaking change` or do something to make sure users see it 🔥 . Thanks!